frontend/src/metabase/metadata/components/FieldOrderSidesheet/FieldOrderSidesheet.tsx

```diff
@@ -14,6 +14,7 @@ import Tables from "metabase/entities/tables";
 import { useDispatch } from "metabase/lib/redux";
 import { PLUGIN_FEATURE_LEVEL_PERMISSIONS } from "metabase/plugins";
 import { Flex } from "metabase/ui";
+// eslint-disable-next-line no-restricted-imports
```

Comment:
This component will be removed when [the project](https://linear.app/metabase/project/up-level-admin-metadata-editing-0399213bee40/issues) is done.